### ISSUE TITLE
Remove code and files related to Conda requirements

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#2358]
These requirements are no more needed since we moved to py3k.

**Comment by gmotzespina** [comment]
@jcohenadad Does this code apply as conda requirement? 

 ```
# run conda installer
cmd="bash $TMP_DIR/miniconda.sh -p $SCT_DIR/$PYTHON_DIR -b -f"
echo ">> "$cmd
$cmd
ret_code_conda=$?
if [[ $ret_code_conda != 0 ]]; then
  echo -e Conda did not installed properly with return code $ret_code_conda
  exit $ret_code_conda
else
  echo -e Conda installation successful
fi

# create py3.6 venv (for Keras/TF compatibility with Centos7, see issue #2270)
yes | python/bin/conda create -n venv_sct python=3.6

# activate miniconda
source python/etc/profile.d/conda.sh
conda activate venv_sct

#  conda install --yes anaconda-client
#  conda config --add channels conda-forge
```

Just want to double check that I won't remove something I shouldn't

**Comment by jcohenadad** [comment]
no don't remove! It's the core of SCT 😉
unless i am missing something, the only file to remove is install/requirements/requirementsConda.txt

(double check-- i might be missing other stuff..)

**Comment by jcohenadad** [comment]
Fixed by #2365
